src/sdk/Audio/AudioOutputFormat.ts

```diff
@@ -532,6 +532,7 @@ export class AudioOutputFormatImpl extends AudioStreamFormatImpl {
     public updateHeader(audioLength: number): void {
         if (this.priHasHeader) {
             const view = new DataView(this.privHeader);
+            view.setUint32(4, audioLength + this.privHeader.byteLength, true);
```

Comment:
[This doc](http://soundfile.sapp.org/doc/WaveFormat/) says it should be `36 + audioLenght` while `this.privHeader.byteLength` is 44

Reply:
hmm, more accurate, it should be `this.privHeader.byteLength - 8 + audioLenght`

Reply:
Yes agree with you @yulin-li - it's mentioned that later one is more precise. I checked size variation by converting .wav to .wav with different engines. That's only possible how you're calculating size.